piksi_tools/serial_link.py

```diff
@@ -107,6 +110,20 @@ def get_logger(use_log=False, filename=LOG_FILENAME):
   print "Logging at %s." % filename
   return JSONLogger(filename)
 
+def get_append_logger(filename):
```

Comment:
In the past I've done it outside of the language with `logrotate` see [logrotate8](http://linuxcommand.org/man_pages/logrotate8.html).